### PR TITLE
passwdqc 2.0.3

### PR DESCRIPTION
**Pull request by p-linnane**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by p-linnane** [comment]
I am not sure why, but our usual Xcode 14.3 workaround doesn't work here. This was the only format that would allow the build to succeed.

File: Formula/passwdqc.rb [hunk 1]
```diff
@@ -32,9 +31,11 @@ def install
     # https://github.com/openwall/passwdqc/issues/15
     inreplace "passwdqc_filter.h", "<endian.h>", "<machine/endian.h>" if OS.mac?
 
+    # CFLAGS arg is a workaround for Xcode 14.3
     args = %W[
       BINDIR=#{bin}
       CC=#{ENV.cc}
+      CFLAGS=-Wno-implicit-function-declaration
```
Comment (carlocab) [inline Formula/passwdqc.rb:38]: Can we add this to `args` only for Xcode 14.3?

File: Formula/passwdqc.rb [hunk 2]
```diff
@@ -49,6 +48,9 @@ def install
       "SECUREDIR=#{prefix}/pam"
     end
 
+    # Workaround for Xcode 14.3
+    args << ("CFLAGS=-Wno-implicit-function-declaration" if DevelopmentTools.clang_build_version >= 1403)
```
Comment (carlocab) [inline Formula/passwdqc.rb:52]:
```suggestion
    # Workaround for Xcode 14.3
    ENV.append_to_cflags "-Wno-implicit-function-declaration" if DevelopmentTools.clang_build_version == 1403
    args << "CFLAGS=#{ENV.cflags}" if ENV.cflags.present?
```
Let's also report the implicit function declaration failure upstream.

**Review by carlocab — APPROVED** [review]

File: Formula/passwdqc.rb (hunk 2 above)
Comment (p-linnane) [reply Formula/passwdqc.rb]: Upstream issue: https://github.com/openwall/passwdqc/issues/31

**Pull request merged**